vc_assistance/indico_vc_assistance/controllers.py

```diff
@@ -0,0 +1,46 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2018 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from collections import OrderedDict
+from operator import itemgetter
+
+from flask import request, session
+from werkzeug.exceptions import Forbidden
+
+from indico.util.date_time import as_utc, get_day_end, get_day_start
+from indico.util.struct.iterables import group_list
+from indico.web.flask.util import url_for
+from indico.web.rh import RHProtected
+
+from indico_vc_assistance.forms import RequestListFilterForm
+from indico_vc_assistance.util import find_requests, is_vc_support
+from indico_vc_assistance.views import WPVCAssistance
+
+
+class RHRequestList(RHProtected):
+    """Provides a list of webcast/recording requests"""
+
+    def _check_access(self):
+        RHProtected._check_access(self)
+        if not is_vc_support(session.user):
+            raise Forbidden
+
+    def _process(self):
+        form = RequestListFilterForm(request.args, csrf_enabled=False)
+        results = None
+        if request.args and form.validate():
```

Comment:
`form.validate_on_submit()` and override `is_submitted()` to return e.g. `bool(request.args)`